src/pages/Popup/Popup.jsx

```diff
@@ -1,56 +1,96 @@
-import React, {useEffect, useState} from 'react';
+import React, { useEffect, useState } from 'react';
 import { createQAUrl } from '../Content/utiles/utiles.js';
 import './Popup.css';
 
 //Inject the content-script to tab
-function injectContentScript(tabId){
-    chrome.scripting.executeScript({
-      files: ['contentScript.bundle.js'],
-      target: { tabId },
-    });
+function injectContentScript(tabId) {
+  chrome.scripting.executeScript({
+    files: ['contentScript.bundle.js'],
+    target: { tabId },
+  });
 }
 
+
 const Popup = () => {
 
-  const [status, setStatus] = useState("initializing");
+  const [status, setStatus] = useState('initializing');
   const [url, setUrl] = useState();
+  const [wixSite, setWixSite] = useState(false);
+  const [jiraLogIn, setJiraLogIn] = useState(false);
+
+  //Set jiraLogIn to true if the user is logged-in in JIRA
+  useEffect(() => {
+    var port = chrome.runtime.connect({
+      name: 'Sample Communication',
+    });
+    port.postMessage('Hi BackGround');
+    port.onMessage.addListener(function (msg) {
+      setJiraLogIn(msg.result);
```

Comment:
We should add a type to the message and only update the log in status when the message type is login